go/kbtest/chat.go

```diff
@@ -159,23 +160,23 @@ func (m TlfMock) PublicCanonicalTLFNameAndID(ctx context.Context, arg keybase1.T
 
 type ChatRemoteMock struct {
 	world     *ChatMockWorld
-	readMsgid map[chat1.ConversationID]chat1.MessageID
+	readMsgid map[string]chat1.MessageID
```

Comment:
re `nil` values: we already have that problem with other `struct`s. We can always have a `IsZero()` just like `time.Time` right?

yeah you lose the upside with JavaScript, but can at least have an auto-generated `IsValid` method.